packages/python-packages/apiview-copilot/prompts/guidelines_review_judge.prompty

```diff
@@ -12,18 +13,17 @@ model:
     azure_deployment: o3-mini
     api_version: 2025-01-01-preview
   parameters:
-    stop: []
     frequency_penalty: 0
     presence_penalty: 0
     max_completion_tokens: 20000
     reasoning_effort: "high"
-    response_format: ${file:general_review_result_schema.json}
+    response_format: ${file:review_result_schema.json}
 ---
 system: |
-  You are a judge that reviews API design feedback to ensure it complies with guidelines and exceptions. Your role is to filter out any improvements that contradict the established rules. You will receive:
+  You are a judge that reviews API design feedback to ensure it complies with guidelines. Your role is to filter out any violations that contradict the established rules. You will receive:
   1. The original APIView content
   2. The initial review results
-  3. The guidelines and exceptions that must be followed
+  3. The guidelines that must be followed
```

Comment:
Do we need this judge prompt? Or is this the one you said you were going to play more with tomorrow?

Reply:
Yeah, the two judge prompts aren't wired up. This PR is pretty big, so I might defer it to a separate PR since most of the value is added by the generic and filter prompts. I'll probably leave the judge prompts in though so it's easy to resume.

Reply:
Actually, to keep things clean I removed the judge prompts. I can always get them back from git history for the next PR